ccpp/data/GFS_typedefs.meta

```diff
@@ -2411,6 +2457,126 @@
   type = real
   kind = kind_phys
   active = (control_for_microphysics_scheme == identifier_for_thompson_microphysics_scheme .and. flag_for_aerosol_physics)
+[emdust]
+  standard_name = emission_of_dust_for_smoke
```

Comment:
Is this assumed to be at the surface? If so, how about `surface_dust_emission`? Why add "for_smoke"

Reply:
That differentiates it from emissions that come from the chemistry model.

Reply:
OK, so would `surface_dust_emission_for_smoke` work?